package.json

```diff
@@ -9,8 +9,15 @@
     "start": "next start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "lint": "eslint . --ext=tsx,ts",
-    "test": "yarn lint"
+    "lint": "eslint . --ext=tsx,ts,js,jsx",
+    "test": "is-ci \"test:coverage\" \"test:watch\"",
+    "test:coverage": "jest --coverage",
```

Comment:
should we have a coverage set now?

Reply:
what exactly do you mean by coverage set?

Reply:
Do @you mean coverage threshold? if so, yes it's in `jest.config.js`